<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="dg_intro_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>650</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>220</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dg_video">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>271</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dg_register">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>950</y>
     <width>322</width>
     <height>152</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>322</width>
     <height>152</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>322</width>
     <height>152</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dg_back">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>950</y>
     <width>322</width>
     <height>152</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>322</width>
     <height>152</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>322</width>
     <height>152</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>331</width>
     <height>569</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>433</width>
        <height>71</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 22pt &quot;나눔스퀘어&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;전화번호 뒷자리(4자)를&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;입력해주세요.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dg_cur_num">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>71</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;나눔스퀘어&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_clr">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_0">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>97</width>
          <height>97</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="plz_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>1366</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>768</width>
     <height>1366</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>plz_bg</zorder>
  <zorder>dg_intro_label</zorder>
  <zorder>dg_video</zorder>
  <zorder>dg_register</zorder>
  <zorder>dg_back</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clr</sender>
   <signal>clicked()</signal>
   <receiver>dg_cur_num</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>840</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
